src/back/spv/block.rs

```diff
@@ -37,6 +37,28 @@ enum ExpressionPointer {
     },
 }
 
+/// The termination statement to be added to the end of the block
+pub enum BlockExit {
+    /// Generates an OpReturn (void return)
+    Return,
+    /// Generates an OpBranch to the specified block
+    Branch {
+        /// The branch target block
+        block_id: Word,
```

Comment:
Suggestion: I would call this `target`. It makes the various construction sites in `write_block` read better.